sympy/matrices/expressions/fourier.py

```diff
@@ -5,14 +5,41 @@
 from sympy import S, I, sqrt, exp
 
 class DFT(MatrixExpr):
-    """ Discrete Fourier Transform """
+    """ Discrete Fourier Transform
+
+    DFT creates the matrix associated with the Discrete Fourier Transform of
+    size n in the standard basis.
+
+    Examples
+    ========
+    >>> from sympy.core.symbol import symbols
+    >>> from sympy.matrices.expressions.fourier import DFT
+    >>> M = DFT(4)
+    >>> N.as_mutable().shape
+    (4, 4)
+    >>> M.as_mutable()
+    Matrix([
+    [1/2,  1/2,  1/2,  1/2],
+    [1/2, -I/2, -1/2,  I/2],
+    [1/2, -1/2,  1/2, -1/2],
+    [1/2,  I/2, -1/2, -I/2]])
+    >>> N, i, j = symbols('N, i, j')
+    >>> DFT(N)[i, j] == DFT(N)[j, i]
+    True
+    See also:
+    ============
+
+    sympy.discrete
+    https://en.wikipedia.org/wiki/DFT_matrix
+"""
```

Comment:
This should be added under `References` section as:
```
References
==========

.. [1] https://en.wikipedia.org/wiki/DFT_matrix
```

Reply:
Oh I see.